docs/sre-guide/support/reset-hub.md

```diff
@@ -0,0 +1,109 @@
+# Reset a Hub
+
+Some hubs are used "seasonally", and need to be "reset" after the end of the workshop /
+event. This helps save on disk space as well as keep the admin interface clean.
+
+This is a *destructive* action, so should only be done in consultation with the
+community champion. Let them know what's going to happen, and await confirmation
+before performing the action. You can use the following template.
+
+```{note}
+We are going to "reset" the hub. This would do the following:
+
+1. Remove **all** user home directories. The *shared* directories would not be
+   touched, but you can clean them up yourself if you need.
+
+2. Remove all the users from the admin view, except 2i2c engineers and those explicitly
+   marked as admins via our configuration. Those are:
+
+   {{ list of admin users from the config.yaml file for this hub}}
+
+This will also forcibly log and stop the server of any user running at that time.
+
+Let us know if this sounds ok, and we will proceed!
+```
+
+Once they respond affirmatively, you can begin.
+
+## Delete all home directories
+
+This clears up space and reduces their cloud costs.
+
+1. Get a shell with the home directories mounted
+
+   ``b
+1. Delete *all* home directories.
+
+   ```bash
+   deployer exec homes <cluster-name> <hub-name>
+   ```
```

Comment:
```suggestion
1. Get a shell with the home directories mounted

   ```bash
   deployer exec homes <cluster-name> <hub-name>
   ```
```

Maybe this is the correct thing, as I see there is a check for `shared` directories below.